.github/workflows/go.yml

```diff
@@ -0,0 +1,89 @@
+name: Go
+
+on:
+  push:
+    branches: [ "v2" ]
```

Comment:
We should only build on tags on the main branch (which is `v2` in this case) and pull requests against it. So this line is not needed.

Reply:
ok